### PR TITLE
Cl league stats

### PR DESCRIPTION
**Pull request by cheeleertr**
Add LeagueStats methods and spec/file. Add spec/files for Team, Games, GameTeams.

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'spec_helper.rb'
+
+RSpec.describe Game do
+  it "can initialize" do
```
Comment (jdmchugh111) [inline spec/game_spec.rb:4]: thanks again for writing these class specs!

**Review by JCL461437 — APPROVED** [review]
Great TDD. Commit for test and then separate one for method. Really easy to follow. Great work Chee!

File: lib/league_stats.rb [hunk 2]
```diff
@@ -1,4 +1,127 @@
 
 
 module LeagueStats
+  def count_of_teams
```
Comment (jdmchugh111) [inline lib/league_stats.rb:4]: these look great!

**Comment by LancePants97** [comment]
Great job with this code and those commits! Love to see it.

**Review by LancePants97 — APPROVED** [review]

**Review by jdmchugh111 — APPROVED** [review]

**Pull request merged**